### ISSUE TITLE
Proposal to add impacted service category as an additional information in the output formats supported by prowler

### ISSUE THREAD

**Issue by rams3sh** [prowler-cloud/prowler#577]
**What is the issue ?** 
Currently I use prowler results (including historical results) of companies I consult in creating trends of compliance of their accounts. As of now , due to lack of information on services for the findings in prowler output, I use a fuzzy logic of terms to be present in a title to confirm the service the finding corresponds to. This helps me creating a AWS service specific trend.

I consume csv export of prowler to create a trend and timeline chart.

**Current State ?**
Currently prowler gives the following info (taken from csv export of prowler scan) :-
```
PROFILE
ACCOUNT_NUM
REGION
TITLE_ID
RESULT
SCORED
LEVEL
TITLE_TEXT
NOTES
```

**Expected output**
I would like prowler to provide an additional field apart from the already existing fields i.e SERVICE_CATEGORY.

Example : For check `[extra765] Check if ECR image scan on push is enabled (Not Scored) (Not part of CIS benchmark)`, it would be nice to add the findings along with the SERVICE_CATEGORY as `AWS ECR`.

**Comment by marcjay** [comment]
Hi @rams3sh 

Have you seen the resource type output of the `json-asff` output mode? That might give you what you're looking for. These are named according to the taxonomy at https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-findings-format.html#securityhub-findings-format-attributes

Example command to output these values for some of the checks:

```
$ ./prowler -M json-asff | jq -r '.Title, .Resources[0].Type'
[check11] Avoid the use of the root account (Scored)
AwsAccount
[check12] Ensure multi-factor authentication (MFA) is enabled for all IAM users that have a console password (Scored)
AwsIamUser
[check13] Ensure credentials unused for 90 days or greater are disabled (Scored)
AwsIamUser
[check14] Ensure access keys are rotated every 90 days or less (Scored)
AwsIamUser
[check15] Ensure IAM password policy requires at least one uppercase letter (Scored)
AwsAccount
[check16] Ensure IAM password policy require at least one lowercase letter (Scored)
AwsAccount
[check17] Ensure IAM password policy require at least one symbol (Scored)
AwsAccount
[check18] Ensure IAM password policy require at least one number (Scored)
AwsAccount
[check19] Ensure IAM password policy requires minimum length of 14 or greater (Scored)
AwsAccount
[check110] Ensure IAM password policy prevents password reuse: 24 or greater (Scored)
AwsAccount
[check111] Ensure IAM password policy expires passwords within 90 days or less (Scored)
AwsAccount
[check112] Ensure no root account access key exists (Scored)
AwsAccount
[check112] Ensure no root account access key exists (Scored)
AwsAccount
[check113] Ensure MFA is enabled for the root account (Scored)
AwsAccount
[check114] Ensure hardware MFA is enabled for the root account (Scored)
AwsAccount
[check116] Ensure IAM policies are attached only to groups or roles (Scored)
AwsIamUser
[check116] Ensure IAM policies are attached only to groups or roles (Scored)
AwsIamUser
[check120] Ensure a support role has been created to manage incidents with AWS Support (Scored)
AwsIamRole
[check121] Do not setup access keys during initial user setup for all IAM users that have a console password (Not Scored)
AwsIamUser
[check122] Ensure IAM policies that allow full "*:*" administrative privileges are not created (Scored)
AwsIamPolicy
[extra774] Ensure credentials unused for 30 days or greater are disabled
AwsIamUser
[check21] Ensure CloudTrail is enabled in all regions (Scored)
AwsCloudTrailTrail
[check23] Ensure the S3 bucket CloudTrail logs to is not publicly accessible (Scored)
AwsS3Bucket
[check24] Ensure CloudTrail trails are integrated with CloudWatch Logs (Scored)
AwsCloudTrailTrail
[check25] Ensure AWS Config is enabled in all regions (Scored)
AwsAccount
[check26] Ensure S3 bucket access logging is enabled on the CloudTrail S3 bucket (Scored)
AwsS3Bucket
[check29] Ensure VPC Flow Logging is Enabled in all VPCs (Scored)
AwsEc2Vpc
```
Maybe you can use this format for output? Else perhaps your request is for these values to be added as a final column in the CSV output

**Comment by rams3sh** [comment]
@marcjay , I checked the one mentioned by you,  however , these do not apply for all the checks like one below :-

`For check [check19] Ensure IAM password policy requires minimum length of 14 or greater (Scored)`

The above finding corresponds to [service category](https://github.com/toniblyx/prowler/issues/578 ) of `IAM` with resource type as `AWS Account` which is not part of current setup.

This proposal is aimed at all checks. And also considering it across all formats rather than csv alone, would be a better scalable option.

This issue is in similar lines with https://github.com/toniblyx/prowler/issues/578 

I think this issue will provide more context across all rules across all formats in general.

**Comment by toniblyx** [comment]
This has been already implemented using `servicename` in outputs since v2.5. ASFF doesn't include it OOTB but has added that attribute in the title.
